programs/vote/src/vote_instruction.rs

```diff
@@ -222,6 +229,10 @@ pub fn process_instruction(
             let to = next_keyed_account(keyed_accounts)?;
             vote_state::withdraw(me, lamports, to, &signers)
         }
+        VoteInstruction::SlashLockouts(tx) => {
+            let slot_hashes = SlotHashes::from_keyed_account(next_keyed_account(keyed_accounts)?)?;
+            vote_state::slash_state(me, &slot_hashes, program_id, &tx)
```

Comment:
We only keep 512 slot hashes around, if somebody submits for slashing a much earlier vote on the same fork, I think it will show up in the diff computed in `slashable_slots()`